web/src/components/vis/PcaPage/PcaPage.vue

```diff
@@ -123,136 +46,225 @@ export default defineComponent({
       return pcaReady && loadingsReady;
     });
 
-    watch(pcXval, (newPcXval) => {
+    watch(() => plot.pcXval, (newPcXval) => {
```

Comment:
Why is this function necessary?

Reply:
I think I was being defensive. Documentation suggests that `reactive` creates deep copies, so presumably `plot.pcXval` is a `ref`, but maybe it wasn't.

Reply:
I'm not sure I follow, sorry.  `reactive` should indeed create deep copies during initialization, but I don't understand how that relates to this wrapper.  `plot.pcXVal` is a reactive proxy object, but it is not a `Ref` type.